### ISSUE TITLE
Change text elements to screen reader friendly HTML

### ISSUE THREAD

**Issue by MichaelZhao21** [MichaelZhao21/tams-club-cal#198]
### Is your feature request related to a problem? Please describe.
A lot of the old elements (code written earlier on) has text elements stored in divs rather than `p` or `h1`, `h2`, etc elements. This should be standard as modern screen readers rely on this to parse website text.

### Describe the solution you'd like
I would like all text to be in either `p` or `h1`, `h2`, etc elements.

### Describe alternatives you've considered
N/A

### Additional context
Same as #37

**Comment by gbudjeakp** [comment]
I would like to work on this issue if possible
